gulpfile.js

```diff
@@ -56,10 +56,10 @@ let contains = require('gulp-contains');
 // exports.logRootFiles   = sb.logRootFiles;
 // exports.lintSamples    = sb.lintSamples;
 
-// exports.logRoutes = logRoutes = gulp.series(
-    // sb.getSamples,
-    // sb.logRoutes,
-// );
+exports.logRoutes = logRoutes = gulp.series(
+    sb.getSamples,
+    sb.logRoutes,
```

Comment:
add logging of sample.SampleFileName in the same console.log() in sb.logRoutes()

Reply:
Done

for (const sample of component.Samples) {
                console.log('' + sample.SampleRoute + '/' + sample.SampleFileName + ' === ' + sample.SampleDisplayName);
            }